lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -312,4 +313,33 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
             return bucket.count >= mincount;
         };
     }
+
+    // remove once [SEARCH-1487] issue is fixed
```

Comment:
please remove this comment and create an issue on JIRA for ADF or ACA if you don't  have one

Reply:
removed & I've created Jira issue https://issues.alfresco.com/jira/browse/ADF-4144 Thanks, Eugenio.